src/model-helpers/Document.js

```diff
@@ -0,0 +1,193 @@
+    .addStringNoLocale(RDF_PREDICATES.gender, dlData.DBC)
+    .addStringNoLocale(RDF_PREDICATES.Eye, dlData.DAY)
+    .addInteger(RDF_PREDICATES.height, Number(dlData.DAU))
+    .addStringNoLocale(RDF_PREDICATES.streetAddress, dlData.DAG)
+    .addStringNoLocale(RDF_PREDICATES.City, dlData.DAI)
+    .addStringNoLocale(RDF_PREDICATES.State, dlData.DAJ)
+    .addStringNoLocale(RDF_PREDICATES.postalCode, dlData.DAK)
+    .addStringNoLocale(RDF_PREDICATES.identifier, dlData.DAQ)
+    .addStringNoLocale(RDF_PREDICATES.identifier, dlData.DCF)
+    .addStringNoLocale(RDF_PREDICATES.Country, dlData.DCG)
+    .addStringNoLocale(RDF_PREDICATES.additionalName, dlData.DDE)
+    .addStringNoLocale(RDF_PREDICATES.additionalName, dlData.DDF)
+    .addStringNoLocale(RDF_PREDICATES.additionalName, dlData.DDG)
+    .build();
+};
+
+/**
+ * Function that generates checksum for uploaded file
+ *
+ * @memberof utils
+ * @function createFileChecksum
+ * @param {fileObjectType} file - Object containing information about file
+ * from form submission (see {@link fileObjectType})
+ * @returns {Promise<WordArray>} Promise - Generates checksum for uploaded file
+ * using the SHA256 algorithm
+ */
+const createFileChecksum = async (file) => {
+  const text = await file.text(); // only hash the first megabyte
+  return sha256(text);
+};
+
+const addAdditionalInfo = (thing, fileObject) => {
+  if (fileObject.type === "Driver's License") {
+    return thing.addUrl(RDF_PREDICATES.about, createDriversLicenseTtlFile(fileObject));
+  }
+  return thing;
+};
+
+export const docDescToThing = async (docDesc, documentUrl, file) => {
+  const checksum = await createFileChecksum(file);
+
+  let thing = buildThing(createThing({ name: docDesc.name }))
+    .addDate(RDF_PREDICATES.uploadDate, new Date())
+    .addStringNoLocale(RDF_PREDICATES.name, docDesc.name)
+    .addStringNoLocale(RDF_PREDICATES.identifier, docDesc.type)
+    .addDate(RDF_PREDICATES.endDate, docDesc.date)
+    .addStringNoLocale(RDF_PREDICATES.sha256, checksum)
+    .addStringNoLocale(RDF_PREDICATES.description, docDesc.description)
+    .addUrl(RDF_PREDICATES.url, documentUrl);
+
+  thing = addAdditionalInfo(thing, docDesc);
+  return thing.build();
+};
+
+export const parseDocument = (documentThing) => {
+  const uploadDate = getDate(documentThing, RDF_PREDICATES.uploadDate);
+  const name = getStringNoLocale(documentThing, RDF_PREDICATES.name);
+  const type = getStringNoLocale(documentThing, RDF_PREDICATES.identifier);
+  const endDate = getDate(documentThing, RDF_PREDICATES.endDate);
+  const checksum = getStringNoLocale(documentThing, RDF_PREDICATES.sha256);
+  const description = getStringNoLocale(documentThing, RDF_PREDICATES.description);
+  const fileUrl = getUrl(documentThing, RDF_PREDICATES.URL);
+  return { uploadDate, name, type, endDate, checksum, description, fileUrl };
+};
+
+const deleteRecursively = async (dataset, options) => {
```

Comment:
Should move this function right above deleteDocument. Like what's done with addAdditionalInfo and docDescToThing